specs/create_safe/02_user_interface.rst

```diff
@@ -0,0 +1,367 @@
+* Zeplin link iOS: https://zpl.io/VOP3pP1
+
+
+
+Recovery phrase view
+--------------------------
+
+Showing the recovery phrase so the user can write it down.
+
+.. raw:: html
+
+  <table>
+    <tr>
+      <td>
+        <img src="screens/android/create_safe_recovery_phrase_view.png" width="320px"/>
+      </td>
+      <td>
+        <img src="screens/ios/create_safe_recovery_phrase_view.png" width="320px"/>
+      </td>
+    </tr>
+  </table>
+
+* Zeplin link Android https://zpl.io/awrk6jJ
+* Zeplin link iOS: https://zpl.io/2yOW80p
+
+
+
+Recovery phrase confirm
+--------------------------
+
+Prompting the user to confirm the recovery phrase
+
+.. raw:: html
+
+  <table>
+    <tr>
+      <td>
+        <img src="screens/android/create_safe_recovery_phrase_confirm_empty.png" width="320px"/>
+      </td>
+      <td>
+        <img src="screens/ios/create_safe_recovery_phrase_confirm_empty.png" width="320px"/>
+      </td>
+    </tr>
+  </table>
+
+* Zeplin link Android https://zpl.io/bzAvEM8
+* Zeplin link iOS: https://zpl.io/bJ7z83n
+
+
+Recovery phrase confirm - 1 word filled
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First word filled
+
+.. raw:: html
+
+  <table>
+    <tr>
+      <td>
+        <img src="screens/android/create_safe_recovery_phrase_confirm_1.png" width="320px"/>
+      </td>
+      <td>
+        <img src="screens/ios/create_safe_recovery_phrase_confirm_1.png" width="320px"/>
+      </td>
+    </tr>
+  </table>
+
+* Zeplin link Android https://zpl.io/bldv5W1
+* Zeplin link iOS: https://zpl.io/beDknjB
+
+
+Recovery phrase confirm - All filled
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All words filled
+
+.. raw:: html
+
+  <table>
+    <tr>
+      <td>
+        <img src="screens/android/create_safe_recovery_phrase_confirm_all.png" width="320px"/>
+      </td>
+      <td>
+        <img src="screens/ios/create_safe_recovery_phrase_confirm_all.png" width="320px"/>
+      </td>
+    </tr>
+  </table>
+
+* Zeplin link Android https://zpl.io/bJ9wy3E
+* Zeplin link iOS: https://zpl.io/V1N9J1m
+
+
+Recovery phrase confirm - Failed
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+User has entered the wrong order
+
+.. raw:: html
+
+  <table>
+    <tr>
+      <td>
+        <img src="screens/android/create_safe_recovery_phrase_confirm_fail.png" width="320px"/>
+      </td>
+      <td>
+        <img src="screens/ios/create_safe_recovery_phrase_confirm_fail.png" width="320px"/>
+      </td>
+    </tr>
+  </table>
+
+* Zeplin link Android https://zpl.io/2v7QPJv
+* Zeplin link iOS: https://zpl.io/aw4j4r1
+
+
+Deposit
+--------
+
+Waiting for the user to deposit funds.
+
+.. raw:: html
+
+  <table>
+    <tr>
+      <td>
+        <img src="screens/android/create_safe_deposit.png" width="320px"/>
+      </td>
+      <td>
+        <img src="screens/ios/create_safe_deposit.png" width="320px"/>
+      </td>
+    </tr>
+  </table>
+
+* Zeplin link Android https://zpl.io/adMJrwp
+* Zeplin link iOS: https://zpl.io/bLP3GGm
```

Comment:
What is the refresh button doing? It is not part of the specs

Reply:
Afaik the funding status of the Safe is checked. Added a note in the specs